project/Dependencies.scala

```diff
@@ -18,6 +18,7 @@ object Dependencies {
   val slf4jVersion = "1.7.25"
   val scalaXmlVersion = "1.0.6"
   val aeronVersion = "1.7.0"
+  val jacksonVersion = "2.9.4"
```

Comment:
Why not the latest version of jackson? Lot of CVEs were fixed since, including active that are not released yet: 
https://github.com/FasterXML/jackson-databind/issues?utf8=%E2%9C%93&q=is%3Aissue+CVE

Reply:
this PR has been closed for a long time, that why. For our migration, we did a local copy of this PR that used up-to-date jackson.

Reply:
For what that's worth, I am planning for `2.9.7` in relatively near future: it does include couple more CVE fixes as listed above, but probably most important here it is possible to hold on to release if there are WIP issues to get in that would help.

Reply:
updated to latest